scenes/quests/lore_quests/quest_001/2_ink_combat/components/ink_combat_example.tscn

```diff
@@ -1,7 +1,7 @@
 [gd_scene load_steps=12 format=4 uid="uid://512qlyin7f7m"]
 
 [ext_resource type="Script" uid="uid://dx4aqh50o2kro" path="res://scenes/quests/lore_quests/quest_001/2_ink_combat/components/ink_combat_example.gd" id="1_3llvl"]
-[ext_resource type="Script" uid="uid://cp54mgi54nywo" path="res://scenes/quests/lore_quests/quest_001/2_ink_combat/components/fill_game_logic.gd" id="3_gntf4"]
+[ext_resource type="Script" path="res://scenes/quests/lore_quests/quest_001/2_ink_combat/components/fill_game_logic.gd" id="3_gntf4"]
```

Comment:
I don't think you meant to commit this and the hunk below here.

Reply:
Sorry about that!